src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html

```diff
@@ -142,26 +140,8 @@ <h4 *ngIf="displaySubsection(3, 5)">Finding Students</h4>
       <tm-student-result-table [studentTables]="exampleMultipleStudentResultTables" [isActionButtonsEnabled]="false"></tm-student-result-table>
     </tm-example-box>
   </tm-instructor-help-panel>
-  <!--Question-->
-  <tm-instructor-help-panel class="instr-help-qn-last" #question [hidden]="key && !showQuestion.includes(StudentsSectionQuestions.STUDENT_EMAIL)"
-                            [id]="StudentsSectionQuestions.STUDENT_EMAIL"
```

Comment:
Hey Howard, I believe we should be able to safely remove this constant as well, could you help to check?

Reply:
You made a point. It has been removed in the last commit. Thanks for the comment.

Reply:
Let's remove the enum const here as well, I don't believe it's being used anywhere else.

https://github.com/TEAMMATES/teammates/blob/9cc2e8451096b900b405d8c698291adf0e782e09/src/web/app/pages-help/instructor-help-page/instructor-help-students-section/students-section-questions.ts#L25-L28

Reply:
This is updated in the latest commit. One other missing part is also fixed.